### ISSUE TITLE
Is it for long-term maintenance?

### ISSUE THREAD

**Issue by sunxiyuan** [wpmts/wp-cloud-deploy#2]
Do you plan to maintain WPCloudDeploy for the long term? WPCloudDeploy is also a dependency of our product. If you plan to maintain it for the long term, we will submit code to this warehouse

**Comment by eslam-dev** [comment]
@sunxiyuan 
Yes, we will continue to develop WPCloudDeploy and provide updates and maintenance for it.

**Comment by qvdrei** [comment]
That's amazing to hear. I would like to contribute to the project. I have a strategy on how to continue to develop and market. Do you want to hop on a zoom and talk about it?

**Comment by eslam-dev** [comment]
@qvdrei 
Yes, you can contact me to schedule a meeting. Reach out to me via email at [dev@eslamelsherif.com](mailto:dev@eslamelsherif.com) or on LinkedIn

**Comment by JFOC** [comment]
wondering if these having premium features same as wpclouddeploy

**Comment by bajpangosh** [comment]
> Do you plan to maintain WPCloudDeploy for the long term? WPCloudDeploy is also a dependency of our product. If you plan to maintain it for the long term, we will submit code to this warehouse

please  maintain WPCD. with full premium features.

**Comment by eslam-dev** [comment]
@JFOC @bajpangosh 
The main plugin will be supported and developed, and alternative plugins for the premium  will also be developed, and they will be fully open-source.

**Comment by eslam-dev** [comment]
@JFOC @bajpangosh @sunxiyuan @qvdrei 

https://discord.gg/bpBWNrxr
can you join the discord server!

**Comment by sunxiyuan** [comment]
> @JFOC @bajpangosh @sunxiyuan @qvdrei
> 
> https://discord.gg/bpBWNrxr can you join the discord server!你能加入 Discord 服务器吗！

the invitation link has expired

**Comment by eslam-dev** [comment]
@sunxiyuan 
https://discord.gg/cjUzZrsr

**Comment by sunxiyuan** [comment]
> @sunxiyuan https://discord.gg/cjUzZrsr

Please tell me the name of the chat room so that I can search and join it myself. The link you recently shared has expired again

**Comment by eslam-dev** [comment]
channel name ; WPCD developvi
 https://discord.gg/kjhta4xQc2
